todo_app/app.py

```diff
@@ -1,23 +1,81 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item
+from todo_app.todo import Item
 
 from todo_app.flask_config import Config
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
+     
+    url = "https://api.trello.com/1/boards/vwBBuYdA/lists"
+
+    print(os.getenv("TRELLO_API_KEY"))
```

Comment:
To avoid noise, remove print statements once you're happy it works

(And printing sensitive values is something you really want to avoid once it comes time to release your application)